schema.graphql

```diff
@@ -37,6 +37,13 @@ type Transaction @entity{
   escrowAccount: EscrowAccount!
 }
 
+type UnnasignedTransaction @entity{
```

Comment:
Rather than creating this new entity, would it be simpler if you just add an unassignedBalance field in the Sender entity? Or is there a reason why you'd prefer this?

Reply:
that does make sense, didnt thought of that thanks!